network/network.go

```diff
@@ -90,6 +91,12 @@ func (a *BzzAddr) Under() []byte {
 	return a.UAddr
 }
 
+// ShortString prints beginning of the OAddr and UAddr
+// It can be used for id in logging
+func (a *BzzAddr) ShortString() string {
+	return fmt.Sprintf("%s:%s ", hex.EncodeToString(a.OAddr)[:16], hex.EncodeToString(a.UAddr)[:16])
```

Comment:
It would be good to handle nil, empty or incomplete Uaddr and Oaddr vales, to avoid panics.

Reply:
Cool, I thought it might be "good" to panic, just so we see if it is used properly along the codebase.